tests/util/setup-dev-tests.ts

```diff
@@ -324,3 +324,14 @@ export function describeDevMoonbeamAllEthTxTypes(
   describeDevMoonbeam(title + " (EIP1559)", cb, "EIP1559", "moonbase", wasm);
   describeDevMoonbeam(title + " (EIP2930)", cb, "EIP2930", "moonbase", wasm);
 }
+
+export function describeDevMoonbeamAllRuntimes(
```

Comment:
Is this being used anywhere? Some uncommitted file?

Reply:
Yeah, I had started working locally on extending some XCM ts tests to moonriver and moonbeam, but most xcm tests need sudo

Reply:
There is the democracy helper for that but it will get removed once we move to Referenda.
Do you think it would make sense to restore sudo under a feature flag in the runtimes for testing purposes ?
Or to allow the client to override storage for testing (not sure the impact on the test however, specially the tree checks)

Reply:
I'd be interested on the feature flag, the last time I tried I could make it work since I could not make it under the construct_runtime macro..

Reply:
I think it's much simpler to add an RPC method allowing the client to modify the storage, I think we should avoid having to compile a special runtime just for testing.